obspy/io/seiscomp/tests/test_inventory.py

```diff
@@ -140,6 +139,48 @@ def test_compare_upper_level(self):
         for sc3ml, stationxml in zip(stationxml_content, sc3ml_content):
             self.assertEqual(sc3ml, stationxml)
 
+    def test_sc3ml_v010(self):
+        """
+        Test reading SC3ML schema version 0.10
+        """
+        data_file = os.path.join(self.data_dir, "sc3ml_010.xml")
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
The unit tests `test_empty_detph` and `test_sc3ml_v010` fail locally when I run the test with :

```bash
obspy-runtests obspy.io.seiscomp.tests.test_inventory
```

but it works with only `obspy-runtests`. Maybe add a `warnings.simplefilter("always")` ?

```python
with warnings.catch_warnings(record=True) as w:
    warnings.simplefilter("always")
```